crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space/bootstrap_task.rs

```diff
@@ -703,67 +702,4 @@ mod tests {
             async move { Ok(self.agents.clone()) }.boxed().into()
         }
     }
-
-    struct HostStub {
```

Comment:
I moved this one out for re-use which didn't end up being necessary for this PR because I'm breaking it into two. I don't see any harm in this staying moved because it will be useful on the next PR